maven/lib/dependabot/maven/file_parser/property_value_finder.rb

```diff
@@ -119,14 +134,14 @@ def parent_repository_urls(pom)
           )
         end
 
+        sig { returns(RepositoriesFinder) }
         def repositories_finder
-          @repositories_finder ||=
-            RepositoriesFinder.new(
-              pom_fetcher: @pom_fetcher,
-              dependency_files: dependency_files,
-              credentials: @credentials,
-              evaluate_properties: false
-            )
+          @repositories_finder ||= T.let(Dependabot::Maven::FileParser::RepositoriesFinder.new(
```

Comment:
indentation change?

Reply:
@raj-meka Please run it through **rubocop -A** to see if this autocorrects.  whitespace doesn't seem like a big deal, but that rule should just check for issues which have already been corrected and make the code less likely to have issues when someone else rebases.